pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: extended_text_library
 description: package library for extended_text and extended_text_field,provide common base class.
-version: 10.0.0
+version: 11.0.0
```

Comment:
We need an entry to the CHANGELOG too.

Reply:
Use unrelease in changelog, update version when publish pkg?
Is it better not to upgrade the version in general PR?

Reply:
Well for packages that don't have pushes frequently, we can apply codes as simply as possible. So I'd vote for manual editing.

Reply:
https://github.com/fluttercandies/extended_text_library/pull/39/commits/1a98e62b5bd57731c4c7c8a8f97c49776ac39409